src/pages/New.js

```diff
@@ -21,15 +21,16 @@ const AddCircleIcon = styled(Add)`
 `;
 
 export default function New() {
+  const [title, setTitle] = React.useState("");
+
   function handleAccept() {
     postRestaurant({
-      imgSrc:
-        "https://static.lieferando.de/images/restaurants/de/OPROO0P/logo_465x320.png",
-      title: "Pasta Pronto",
-      categories: ["pizza", "salad", "pasta"],
-      distance: 6,
-      rating: 3.1,
-      description: "Eat italian"
+      imgSrc: "",
+      title: title,
```

Comment:
you changed much more than only the title. this would be an issue if someone else is working on `distance` for example. better only change the title here.